src/Aquifer.API/Endpoints/Resources/Content/Batch/Text/Endpoint.cs

```diff
@@ -46,6 +46,6 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
     public override async Task OnAfterHandleAsync(Request req, List<Response> res, CancellationToken ct)
     {
         const string endpointId = "resources-content-batch-text";
-        await trackingService.TrackAsync(HttpContext, req.Ids, endpointId);
+        await trackingService.TrackAsync(HttpContext, req.Ids, endpointId, cancellationToken: ct);
```

Comment:
I made the resource tracking job conform to the new patterns I introduced.  After this PR merges I'm also planning to move the existing Jobs into the new sub-folders (`Subscribers` and `Managers`).